README.md

```diff
@@ -2,6 +2,124 @@
 
 The repository contains various [Concourse CI](https://concourse-ci.org) pipeline definitions. Most pipeline definitions live with the code they are intended to be used with, however some pipelines do not have code. This space is used to house those pipelines that are code or service independent.
 
+## Testing a local pipeline by example
+
+Let's say we would like to test a pipeline that pushes a new docker hub image whenever a new tag is pushed to a git repository. First, we should take a look at the docker-compose.yml file included in this repo.
+
+```yml
+# docker-compose.yml
+version: '3.7'
+services:
+  concourse-db:
+    image: postgres:12
+    # ...
+
+  concourse:
+    image: concourse/concourse:4.2.2
+    ports: ["8080:8080"]
+    # ...
+
+  docker-registry:
+    restart: always
+    image: registry:2
+    ports: ["5000:5000"]
+
+  git-server:
+    image: tomjw64/git-server
```

Comment:
Could this be transferred over to openstax?

Reply:
I can if people want to use it. I didn't want to litter the openstax org with stuff nobody uses.

Reply:
I see your point... if we're suggesting that people use it for local testing of concourse then it seems like it'd be something that others _would_ use.